tools/video_tools/pyscenedetect_shot_detection.xml

```diff
@@ -0,0 +1,20 @@
+<tool id="PySceneDetect_shot_detection" name="PySceneDetect" version="1.0.0">
+  <description>Runs shot detection on a video</description>
+    <requirements>
+    </requirements>
+  <command detect_errors="exit_code"> $__tool_directory__/pyscenedetect_shot_detection.py $input $threshold $amp_shots $frame_stats</command>
```

Comment:
try to quote all your data inputs

Reply:
I will create a story to update all the tools we added so far with quotes. 

On a related subject, there is a sanitary check attribute for parameters which I turned off for all the hmgm tools we added, due to the fact that the sanitary checking adds some extra chars that interfere with one of the parameter (it's a bit complicated to explain, but in short it relates to the way different shells parses command line params with spaces and quotes and escape chars).